README.md

```diff
@@ -1,5 +1,11 @@
 # Sidero
 
+!!! ATTENTION !!!
+Sidero Labs is no longer actively developing Sidero Metal.
+For an alternative, please see [Omni](https://github.com/siderolabs/omni.git).
+Any and all support will be contributed by the community (including questions in our Slack workspace).
```

Comment:
Unless you have an existing support contract covering Sidero Metal, all support will be provided by the community (including questions in our Slack workspace).